packages/client/src/lobby/history.ts

```diff
@@ -319,16 +323,16 @@ export function drawOtherScores(
 
     // Column 6 - Seed. Chop off the prefix.
     const match = /p\dv\d+s(\d+)/.exec(gameData.seed);
-    let seed: string;
+    let seedNumber: string;
```

Comment:
shouldn't this be called `seedString`

Reply:
oh, i see, because it is a subset that is a number.
can we call it `seedNumberSuffix` or something like that?